language-server/modules/langserver-core/src/test/resources/completion/statement_context/config/elseif_stmt_ctx_config1.json

```diff
@@ -605,19 +605,19 @@
         }
       },
       "sortText": "C",
-      "insertText": "testFunction()",
+      "insertText": "doSomeTask()",
       "insertTextFormat": "Snippet"
     },
     {
       "label": "testInt",
       "kind": "Variable",
-      "detail": "int",
+      "detail": "2",
```

Comment:
In this scenario, for the the `testInt` variable, the type is shown as `2` instead of `int`, which was the earlier behaviour. Refer the following user experience.

<img width="884" alt="Screenshot 2021-05-11 at 09 51 47" src="https://user-images.githubusercontent.com/1329674/117757896-93a96200-b23e-11eb-9aab-1ffcd15fa5dc.png">